package.json

```diff
@@ -36,55 +36,51 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.5.0",
+    "broccoli-asset-rev": "^2.4.5",
     "chai-jquery": "^2.0.0",
     "ember-browserify": "1.1.13",
-    "ember-cli": "~2.11.0",
-    "ember-cli-app-version": "^2.0.1",
+    "ember-cli": "2.12.3",
     "ember-cli-chai": "^0.3.2",
     "ember-cli-code-coverage": "0.3.11",
     "ember-cli-dependency-checker": "^1.3.0",
-    "ember-cli-htmlbars-inline-precompile": "0.3.6",
+    "ember-cli-frost-blueprints": "^1.0.0",
+    "ember-cli-htmlbars-inline-precompile": "0.3.12",
```

Comment:
Re-run the build that was successful before, with no code change, and got build error from latest `ember-cli-htmlbars-inline-precompile` that just released. Pin it to 0.3.12 until it gets fixed.